tests/release/e2e-test-default-with-deployment.go

```diff
@@ -0,0 +1,231 @@
+					Policy: []string{
+						"git::https://github.com/hacbs-contract/ec-policies.git//policy",
+					},
+					Data: []string{
+						"git::https://github.com/hacbs-contract/ec-policies.git//data",
+					},
+				},
+			},
+			Exceptions: &ecp.EnterpriseContractPolicyExceptions{
+				NonBlocking: []string{"tasks", "attestation_task_bundle", "java", "test", "not_useful"},
+			},
+		}
+
+		_, err = framework.CommonController.CreateConfigMap(cm, devNamespace)
+		Expect(err).NotTo(HaveOccurred())
+
+		_, err = framework.ReleaseController.CreateReleasePlan(sourceReleasePlanName, devNamespace, applicationNameDefault, managedNamespace, "")
+		Expect(err).NotTo(HaveOccurred())
+
+		_, err = framework.ReleaseController.CreateReleaseStrategy(releaseStrategyDefaultName, managedNamespace, releasePipelineNameDefault, releasePipelineBundleDefault, releaseStrategyPolicyDefault, releaseStrategyServiceAccountDefault, paramsReleaseStrategy)
+		Expect(err).NotTo(HaveOccurred())
+
+		_, err = framework.GitOpsController.CreateEnvironment(releaseEnvironment, managedNamespace)
+		Expect(err).NotTo(HaveOccurred())
+
+		_, err = framework.ReleaseController.CreateReleasePlanAdmission(targetReleasePlanAdmissionName, devNamespace, applicationNameDefault, managedNamespace, releaseEnvironment, "", releaseStrategyDefaultName)
+		Expect(err).NotTo(HaveOccurred())
+
+		_, err = framework.TektonController.CreateEnterpriseContractPolicy(releaseStrategyPolicyDefault, managedNamespace, defaultEcPolicy)
+		Expect(err).NotTo(HaveOccurred())
+
+		_, err = framework.TektonController.CreatePVCInAccessMode(releasePvcName, managedNamespace, corev1.ReadWriteOnce)
+		Expect(err).NotTo(HaveOccurred())
+
+		_, err = framework.CommonController.CreateServiceAccount(releaseStrategyServiceAccountDefault, managedNamespace, managednamespaceSecret)
+		Expect(err).NotTo(HaveOccurred())
+
+		_, err = framework.CommonController.CreateRole(roleName, managedNamespace, roleRules)
+		Expect(err).NotTo(HaveOccurred())
+
+		_, err = framework.CommonController.CreateRoleBinding("role-relase-service-account-binding", managedNamespace, "ServiceAccount", roleName, "Role", "role-m6-service-account", "rbac.authorization.k8s.io")
+		Expect(err).NotTo(HaveOccurred())
+
+		_, err = framework.HasController.CreateHasApplication(applicationNameDefault, devNamespace)
+		Expect(err).NotTo(HaveOccurred())
+
+		_, err = framework.HasController.CreateComponent(applicationNameDefault, componentName, devNamespace, gitSourceComponentUrl, "", containerImageUrl, "", "", false)
+		Expect(err).NotTo(HaveOccurred())
+
+	})
+
+	AfterAll(func() {
+
+		if !CurrentSpecReport().Failed() {
+			Expect(framework.CommonController.DeleteNamespace(devNamespace)).NotTo(HaveOccurred())
+			Expect(framework.CommonController.DeleteNamespace(managedNamespace)).NotTo(HaveOccurred())
+		}
+	})
+
+	var _ = Describe("Post-release verification", func() {
+
+		It("verifies that in dev namespace will be created a PipelineRun.", func() {
```

Comment:
It doesn't read very well. I would suggest something like: `"verifies that a PipelineRun is created in dev namespace."`

Reply:
fixed